src/js/components/wizard.js

```diff
@@ -0,0 +1,103 @@
+import FluxComponent from 'flummox/component';
+import React from 'react';
+
+
+let Wizard = React.createClass({
+  render() {
+    let steps = this.props.children.map(
+      (step, index) => React.cloneElement(step, {
+        key: index,
+        isActive: index === this.props.activeStep
+      })
+    , this);
```

Comment:
Do you need to set the `this` when you use a `=>` function? I thought `() => {}` was equivalent to `function() {}.bind(this)`.